### DISCUSSION TITLE
Compatibility with unocss

### DISCUSSION THREAD

**Discussion by jbaubree** [dcastil/tailwind-merge#233] [General]
Hi, this is a great plugin.
Is it compatible with [UnoCSS](https://github.com/unocss/unocss)?

**Comment by dcastil** [comment]
Hey @jbaubree! 👋

Thanks!

Yes, you can use tailwind-merge with UnoCSS or any utilities-based class structure, but you need to configure all the classes in conflict for it in tailwind-merge. This might be a lot of work though, e.g. check out the default config I use tell tailwind-merge about all the rules in Tailwind CSS: https://github.com/dcastil/tailwind-merge/blob/v1.12.0/src/lib/default-config.ts

[Here](https://github.com/dcastil/tailwind-merge/blob/v1.12.0/docs/configuration.md) you can read more about how to configure tailwind-merge.
